qiskit/circuit/library/__init__.py

```diff
@@ -126,6 +126,13 @@
    ZGate
    GlobalPhaseGate
 
+
+Utility functions (section TBD)
+================================
+
+.. autofunction:: get_standard_gate_name_mapping
```

Comment:
Qiskit devs: do you have any suggestions on what to name the section and where it makes sense to live in this module page? I don't use the circuit library enough to have an opinion.

Reply:
I put it in the prelude (without a section) as it is the only one (see 18693b7210c857acdfc60ff224092e2739c1ceaf). If, in the future, we get more, we can create a _tooling_ section. what do you think?

Reply:
It looks like this:
![get_standard_gate_name_mapping](https://github.com/user-attachments/assets/f94fd084-b4fc-4b89-a33a-02e6f4124579)

If a native speaker can check the English, we are good to merge this one.

Reply:
> If a native speaker can check the English, we are good to merge this one.

@Eric-Arellano ?